app/models/comment.rb

```diff
@@ -215,6 +215,31 @@ def send_to_moderator
   end
 
   def evaluate_markdown
+    if FeatureFlag.enabled?(:consistent_rendering, FeatureFlag::Actor[user])
+      extracted_evaluate_markdown
+    else
+      original_evaluate_markdown
+    end
+  end
+
+  def processed_content
```

Comment:
Could this and the one below be a private method?

Reply:
:thinking: Aren't they private already? (`private` is on `:193`)

Reply:
Oops sorry @lightalloy. I didn't see the private.